src/OneExplorer/OneStorage.ts

```diff
@@ -246,7 +246,9 @@ export class OneStorage {
     try {
       return roots
         .map((root) =>
-          readdirSyncRecursive(root).filter((val) => val.endsWith(".cfg"))
+          readdirSyncRecursive(root).filter(
+            (val) => extList.some((ext) => val.endsWith(ext))
```

Comment:
Thank you. but **extList** is undefined yet... 😢

Reply:
Oh... I missed It :(  I will append extList :-D